Deprecate AC documentation (#2068)
astronomer/docs: archived_software_docs/astronomer-certified/ac-cve.md

description: "Common Vulnerabilities and Exposures identified within our Astronomer Certified images."
---

> > **Warning**
> > No versions of Astronomer Certified (AC) are currently supported by Astronomer. Astronomer stopped releasing new versions of AC with the release of Apache Airflow 2.4. Astronomer recommends creating all new Deployments with Astro Runtime, as well as migrating existing Deployments from AC to Astro Runtime as soon as your organization is ready. See [Migrate to Runtime](migrate-to-runtime.md) and [Runtime image architecture](runtime-image-architecture.md).

Reviewer: ```suggestion
::: warning

No versions of Astronomer Certified (AC) are currently supported by Astronomer. Astronomer stopped releasing new versions of AC with the release of Apache Airflow 2.4. Astronomer recommends that you create all new Deployments with Astro Runtime and migrate existing Deployments from AC to Astro Runtime as soon as your organization is ready. See [Migrate to Runtime](migrate-to-runtime.md) and [Runtime image architecture](runtime-image-architecture.md).
```

Format?
